### PR TITLE
Fix wrong default value in tiflash config doc

### PR DESCRIPTION
**Pull request by LittleFall**
### What is changed, added or deleted? (Required)

Add `toml` note for config files.

According to TiFlash code, the default value of `max_memory_usage` is 0 instead of 10'000'000'000, so I fix it.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
  need to translate to pingcap/docs.
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ minmax_index_cache_size = 5368709120 # 数据块 min-max 索引的内存 cache 
 [profiles]
 [profiles.default]
     dt_enable_logical_split = true # 存储引擎的 segment 分裂是否使用逻辑分裂。使用逻辑分裂可以减小写放大，提高写入速度，但是会造成一定的空间浪费。默认为 true
-    max_memory_usage = 10000000000 # 单次 coprocessor 查询过程中，对中间数据的内存限制，单位为 byte，默认为 10000000000。如果设置为 0 表示不限制
+    max_memory_usage = 0 # 单次 coprocessor 查询过程中，对中间数据的内存限制，单位为 byte，默认为 0，表示不限制
```
Comment (zanmato1984) [inline tiflash/tiflash-configuration.md:82]: Better add unit in description, i.e. "in bytes".

Comment (LittleFall) [reply tiflash/tiflash-configuration.md:82]: Can't  `单位为 byte` meet the demand?

Comment (zanmato1984) [reply tiflash/tiflash-configuration.md:82]: Sorry, I was blind.

**Review by zanmato1984 — APPROVED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**